hub-config/tasks.json

```diff
@@ -8,17 +8,165 @@
+                                -26,
+                                -25,
+                                -24,
+                                -23,
+                                -22,
+                                -21,
+                                -20,
+                                -19,
+                                -18,
+                                -17,
+                                -16,
+                                -15,
+                                -16,
+                                -15,
+                                -14,
+                                -13,
+                                -12,
+                                -11,
+                                -10,
+                                -9,
+                                -8,
+                                -7,
+                                -6,
+                                -5,
+                                -4,
+                                -3,
+                                -2,
+                                -1,
+                                0,
+                                1,
+                                2,
+                                3,
+                                4,
+                                5,
+                                6,
+                                7 
+                            ]
                         }
                     },
-                    "output_type": {},
-                    "target_metadata":[]
+                    "output_type": {
+                        "mean": {
+                            "output_type_id": {
+                                "required": ["NA"],
+                                "optional": null
+                            },
+                            "value": {
+                                "type": "double",
+                                "minimum": 0,
+                                "maximum": 1
+                            }
+                        },
+                        "sample": {
+                            "output_type_id_params": {
+                                "is_required": false,
+                                "type": "character",
+                                "max_length": 500,
+                                "min_samples_per_task": 96,
+                                "max_samples_per_task": 100,
+                                "compound_taskid_set": ["origin_date", "location", "variant", "horizon"]
+                            },
+                            "value": {
+                                "type": "double",
+                                "minimum": 0,
+                                "maximum": 1
+                            }
+                        }
+                    },
+                    "target_metadata":[
+                        {
+                            "target_id": "variant prop",
+                            "target_name": "variant proportion of all sequences",
```

Comment:
should be something like "frequency of observed variant lineage". the "of all sequences" didn't sound right to me. But we might also want to be more choosy about the "variant lineage" terminology here.